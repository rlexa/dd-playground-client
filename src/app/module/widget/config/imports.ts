import { CommonModule } from '@angular/common';
import { MatCardModule, MatListModule } from '@angular/material';
import { RouterModule } from '@angular/router';

export const imports = [CommonModule, RouterModule, MatCardModule, MatListModule];
