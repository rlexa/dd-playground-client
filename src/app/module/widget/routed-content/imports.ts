import { RouterModule } from '@angular/router';

export const imports = [ RouterModule];
