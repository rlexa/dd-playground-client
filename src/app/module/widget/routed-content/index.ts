export * from './routed-content.component';
export * from './routed-content.module';
