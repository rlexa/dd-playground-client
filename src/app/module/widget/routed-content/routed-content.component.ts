import {ChangeDetectionStrategy, Component} from '@angular/core';

@Component({
  selector: 'app-routed-content',
  templateUrl: './routed-content.component.html',
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class RoutedContentComponent {}
