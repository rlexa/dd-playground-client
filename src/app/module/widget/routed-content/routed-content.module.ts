import { NgModule } from '@angular/core';
import { imports } from './imports';
import { RoutedContentComponent } from './routed-content.component';

@NgModule({ imports, exports: [RoutedContentComponent], declarations: [RoutedContentComponent] })
export class RoutedContentModule { }
