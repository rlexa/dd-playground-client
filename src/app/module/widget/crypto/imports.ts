import { CommonModule } from '@angular/common';
import { FormsModule } from '@angular/forms';
import {
  MatButtonModule,
  MatCardModule,
  MatFormFieldModule,
  MatInputModule,
  MatListModule,
} from '@angular/material';
import { RouterModule } from '@angular/router';
import { CryptoApiModule } from 'app/module/service/crypto-api';
import { SimpleViewModule } from 'app/module/widget/simple-view';
import { FlexboxModule } from 'app/module/directive/flexbox';

export const imports = [
  CommonModule,
  FormsModule,
  RouterModule,
  MatButtonModule,
  MatCardModule,
  MatFormFieldModule,
  MatInputModule,
  MatListModule,
  FlexboxModule,
  CryptoApiModule,
  SimpleViewModule,
];
