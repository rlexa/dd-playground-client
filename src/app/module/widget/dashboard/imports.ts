import { CommonModule } from '@angular/common';
import { MatButtonModule, MatToolbarModule, MatTooltipModule } from '@angular/material';
import { RouterModule } from '@angular/router';
import { IconPipeModule } from 'app/module/pipe/icon';
import { RoutedContentModule } from 'app/module/widget/routed-content';
import { VersionModule } from 'app/module/widget/version';
import { WidgetsModule } from 'app/widgets/widgets.module';
import { FlexboxModule } from 'dd-flexbox';
import 'hammerjs';

export const imports = [
  CommonModule, RouterModule,
  MatButtonModule, MatToolbarModule, MatTooltipModule, FlexboxModule,
  IconPipeModule, VersionModule,
  // remove
  RoutedContentModule, WidgetsModule,
];
