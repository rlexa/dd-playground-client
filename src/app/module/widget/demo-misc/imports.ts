import { CommonModule } from '@angular/common';
import { MatButtonModule, MatCardModule } from '@angular/material';
import { RouterModule } from '@angular/router';
import { GraphskyApiModule } from 'app/module/service/graphsky-api';
import { DiagramPolynomModule } from 'app/module/widget/diagram-polynom';
import { SimpleViewModule } from 'app/module/widget/simple-view';
import { FlexboxModule } from 'app/module/directive/flexbox';

export const imports = [
  CommonModule,
  RouterModule,
  MatButtonModule,
  MatCardModule,
  FlexboxModule,
  DiagramPolynomModule,
  GraphskyApiModule,
  SimpleViewModule,
];
