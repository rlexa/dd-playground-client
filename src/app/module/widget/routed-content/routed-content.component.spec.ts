import {async, TestBed} from '@angular/core/testing';
import {RouterTestingModule} from '@angular/router/testing';
import {imports} from './imports';
import {RoutedContentComponent} from './routed-content.component';

describe('RoutedContentComponent', () => {
  beforeEach(async(() => {
    TestBed.configureTestingModule({
      imports: [RouterTestingModule, ...imports],
      declarations: [RoutedContentComponent],
      providers: [],
    }).compileComponents();
  }));

  test('is created', () => {
    const fixture = TestBed.createComponent(RoutedContentComponent);
    fixture.detectChanges();
    expect(fixture).toMatchSnapshot();
  });
});
