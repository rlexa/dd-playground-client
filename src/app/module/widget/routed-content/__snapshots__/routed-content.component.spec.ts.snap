// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoutedContentComponent is created 1`] = `
<app-routed-content>
  <div
    class="position-absolute match-parent scroll-y-always scroll-performance container"
  >
    <router-outlet />
  </div>
</app-routed-content>
`;
